admin/schema.php

```diff
@@ -884,6 +884,11 @@
 
 # Release marker: 1.3.0
 
+$g_upgrade[210] = array( 'AddColumnSQL', array( db_get_table( 'mantis_custom_field_table' ), "
+	color_background     C(8)		NOTNULL DEFAULT \" '' \",
```

Comment:
Please explain why you set length to 8. I would have expected 7.

Reply:
changed to 7